bagel/mappings.py

```diff
@@ -65,9 +65,8 @@ def get_pipeline_catalog(url: str, path: Path) -> list[dict]:
         return response.json()
     # The JSONDecodeError should catch the case where the file is empty
     except (httpx.HTTPError, json.JSONDecodeError):
-        warnings.warn(
-            f"Unable to download pipeline catalog from {url}.\n"
-            f"Will revert to loading backup from {path}."
+        logger.warning(
+            f"Unable to download pipeline catalog from {url}. Will revert to loading backup from {path}."
```

Comment:
```suggestion
            f"Unable to download pipeline catalog from {url}.\n"
            f"Will revert to loading backup from {path}."
```
The linebreak in the print is useful if the URL is long. The linebreak in the code is useful to keep it readable and easier to change individual lines